index.html

```diff
@@ -5,115 +5,151 @@
     </head>
     <body>
-        <header>
-            <div>
-                <a id="logo" href="index.html"><img src="img/logo.png"/></a>
-                <nav>
-                    <a href="https://mic.am/">HOME<span>Back to home</span></a>
-                    <a href="https://mic.am/training-program/">PRODUCTS<span>What we have for you</span></a>
-                    <a href="https://mic.am/projects/">SERVICES<span>Things we do</span></a>
-                    <a href="https://mic.am/blog/">BLOG<span>Follow our updates</span></a>
-                    <a href="https://mic.am/about-us/">CONTACT<span>Ways to reach us</span></a>
-                </nav>
-            </div>
+        <header class="header">
+            <a class="logoHeader" href="index.html"><img src="img/logo.png"/></a>
+            <nav class="menuHeader">
+                <a href="https://mic.am/"><span class="linkHeader">HOME</span><br /><span class="linkDescription">Back to home</span></a>
+                <a href="https://mic.am/training-program/"><span class="linkHeader">PRODUCTS</span><span class="linkDescription">What we have for you</span class="linkDescription"></a>
+                <a href="https://mic.am/projects/"><span class="linkHeader">SERVICES</span><span class="linkDescription">Things we do</span></a>
+                <a href="https://mic.am/blog/"><span class="linkHeader">BLOG</span><span class="linkDescription">Follow our updates</span></a>
+                <a href="https://mic.am/about-us/"><span class="linkHeader">CONTACT</span><span class="linkDescription">Ways to reach us</span></a>
+            </nav>
         </header>
-        <div id="top-background"></div>
+        <div class="topBackground"></div>
         <main>
-            <div><img id="back-big" src="img/back-big.png" /></div>
-            <h1>Lorem ipsum dolor sit amet, consectetur adipisicing elit</h1>
-            <hr id="hr1" />
-            <div class="article__top">
-                <article class="verev">
-                    <div>
-                        <a class="vernagir" href="index.html">Perfect Logic</a class="vernagir">
-                        <p class="under__a">All you want your website to do.</p class="under">
-                        <img class="img" src="img/beran.png" alt="">
-                        <p class="under__img">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p class="under__img">
-                        <a class="button" href="index.html">
-                            <div class="button__div">
-                                <span>Learn More</span>
-                                <span>&#8674;</span>
-                            </div>
-                        </a>
-                    </div>
+            <div><img class="backBig" src="img/backBig_1.jpg" /></div>
```

Comment:
Do not use partly words for className. backBig doesn't give meaningful idea what this HTML tag is doing.